frameworks/python/cw-2.py

```diff
@@ -1,24 +1,49 @@
 from __future__ import print_function
+import six
 
 class AssertException(Exception):
     pass
 
 
+'''Fix the dreaded Unicode Error Trap'''
+def print(*args, **kwargs):
```

Comment:
This `print` doesn't seem to work when given non-string, e.g., `print(1)` throws
```
TypeError: 'int' object is not iterable
```
in Python 2.7 and
```
TypeError: print() argument after * must be a sequence, not map
```
in Python 3